src/duke/Storage.java

```diff
@@ -0,0 +1,77 @@
+package duke;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class Storage {
+    String filePath;
+    File f;
```

Comment:
Name it File variable as file. f is supposed to be used for scratch variables (i.e. short term use)